sjb/config/test_cases/test_branch_origin_extended_conformance_crio_rpm.yml

```diff
@@ -38,6 +38,36 @@ extensions:
         name = OpenShift Ansible Release from Local Source
         EOR
         sudo cp ./openshift-ansible-local-release.repo /etc/yum.repos.d
+    - type: "script"
+      title: "turn off unnecessary RPM repos"
+      timeout: 300
+      script: |-
+        sudo yum-config-manager --disable centos-paas-sig-openshift-origin\*-rpms
+        sudo yum-config-manager --disable origin-deps-rhel7\* || true
+        sudo yum-config-manager --disable rhel-7-server-ose\* || true
+
+        branch="${PULL_BASE_REF:-"master"}"
+        case "${branch}" in
+          master )
+            # 3.10
+            major=3
+            minor=10
+            ;;
```

Comment:
This is the case I'm most worried about, maintenance wise.  It means when 3.11 comes around, some human needs to remember to edit this.  Is there no, like, ``VERSION`` file or something that could be used?  How about pulling it from ``git describe`` output?

Reply:
idea:

```bash
    tag_version=$(git describe --abbrev | sed -r 's/^openshift-ansible-([0-9\.]+)-.+/\1/')
    major=$(echo $tag_version | cut -d . -f 1)
    minor=$(echo $tag_version | cut -d . -f 2)
```